Mathlib/Algebra/GroupWithZero/Action/Defs.lean

```diff
@@ -423,4 +424,22 @@ lemma smul_eq_zero_iff_eq (a : α) {x : β} : a • x = 0 ↔ x = 0 :=
 lemma smul_ne_zero_iff_ne (a : α) {x : β} : a • x ≠ 0 ↔ x ≠ 0 :=
   not_congr <| smul_eq_zero_iff_eq a
 
-end Group
+end DistribMulAction
+
+section MulDistribMulAction
+
+variable [Group α] [GroupWithZero β] [MulDistribMulAction α β]
+
+instance : SMulZeroClass α β where
+  smul_zero g := not_imp_comm.mp mul_inv_cancel₀ <| by
+    rw [← smul_one g, ← inv_smul_eq_iff, smul_mul', inv_smul_smul, zero_mul]
+    exact zero_ne_one
+
+theorem smul_div₀ (g : α) (x y : β) : g • (x / y) = (g • x) / (g • y) := by
```

Comment:
Sorry for not spotting this previously; I think this should be `smul_div₀'`, to avoid matching `smul_inv₀`